### ISSUE TITLE
Provenance graph breaks when analysis is lost

### ISSUE THREAD

**Issue by flekschas** [refinery-platform/refinery-platform#1098]
Commit: 336b2be21635b8735b7f5c2f0f3f6a384f80fac4
### Steps to reproduce
1. Delete a successful analysis or mess up your vm somehow
2. Go to http://192.168.50.50:8000/data_sets/UUID/#/files/browse
3. Select the provvis
### Observed behavior

Provvis spinner keeps on spinning indefinitely because:

```
provvis-layout.js:374
Uncaught TypeError: Cannot read property 'autoId' of undefined
    (anonymous function)    @   provvis-layout.js:374
    dagreGraphLayout        @   provvis-layout.js:373
    runLayoutPrivate        @   provvis-layout.js:405
    provvisLayout.run       @   provvis-layout.js:448
    (anonymous function)    @   provvis.js:422
    (anonymous function)    @   d3.min.js:1
    t                       @   d3.min.js:1
    u                       @   d3.min.js:1
```
### Expected behavior

The provvis initialization should fail with an error being displayed or the visualization should ignore that analysis.

**tl/dr:**
Stefan identified that issues already, see: https://github.com/parklab/refinery-platform/blob/develop/refinery/ui/source/js/provvis/provvis-init.js#L497-L498. This is just for documentation.

**Comment by jkmarx** [comment]
Encapsulated in larger issue, Ref #1338
